pkg/server/server.go

```diff
@@ -35,6 +35,13 @@ func New(config configuration.Configuration, application application.Application
 	ginRouter.Use(
 		gin.LoggerWithWriter(gin.DefaultWriter, "/api/v1/health"),
 		gin.Recovery(),
+		cors.New(cors.Config{
+			AllowOrigins:     []string{"*"},
```

Comment:
You can use AllowAllOrigins for that too:
```suggestion
			AllowAllOrigins:     true,
```

Reply:
ohhh okay, nice!